src/com/google/cose/Encrypt0Message.java

```diff
@@ -37,6 +37,18 @@ public Builder withProtectedHeaderBytes(byte[] protectedHeaderBytes) {
       return this;
     }
 
+    public Builder withProtectedHeaders(Map protectedHeaders) {
+      if (protectedHeaderBytes != null) {
+        throw new CoseException("Cannot use both withProtectedHeaderBytes and withProtectedHeaders");
```

Comment:
This string is misleading. As it stands, we cannot differentiate if someone called this method twice _or_ `withProtectedHeaderBytes` followed by `withProtectedHeaders`.

Reply:
Maybe say "protected headers may only be added once"?